examples/gno.land/r/demo/markdown_test/markdown.gno

```diff
@@ -175,6 +173,403 @@ If markdown is too limiting, you can just insert your own <strike>crazy</strike>
 <div style='font-family: "Comic Sans MS", "Comic Sans", cursive;'>
 It is a pity, but markdown does **not** work in here for most markdown parsers.
 [Marked] handles it pretty well.
-</div>`
+</div>
+
+Components
+==========
+
+Jumbotron component
+-------------------
+
+A Jumbotron is a layout component used to showcase key messages.
+
+
+:::jumbotron
+
+## Jumbotron title
+
+This is a Jumbotron component you can fill with regular markdown.
+
+:::jumbotron/
+
+
+Stack component
+---------------
+
+Stack is a layout component used to group elements together and apply a space between them. For example, it is usefull with grouped links.
+
+:::stack
+[Stack link 1](/)
+[Stack link 2](/)
+:::stack/
+
+Box component
+-------------
+
+Box is the most abstract component that renders a div element. It is used as a group component.
+
+:::box
+**This content is in a box**
+:::box/
```

Comment:
```suggestion
:::box
**This content is in a box**
:::/box
```

looks more natural to me. what do you think?

Reply:
I do agree with you, I've updated it in that way